examples/get-started/create-react-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Do you need this file specifically for this example? Can this be rolled into the root .gitignore?